### ISSUE TITLE
[Bug]Fool's Stout conditions

### ISSUE THREAD

**Issue by Crysicle** [TrinityCore/TrinityCore#12227]
"Fool's Stout" and perhaps a few other quests associated with it, seem to be missing some conditions. The conditions for these quests below are speculations. These quests need sniffs to fix properly.

The quest "Fool's Stout" rewards an item "Fool's Stout" in case the 1st "Fool's Stout" which is provided for 3 other quests runs out of time(2 hours), however the way the quest is handled doesn't make sense:
1. By abandoning any of the 3 quests and retaking them, you get back "Fool's Stout" with a full timer, thus no reason for the quest "Fool's Stout" to even exist;
2. The quest is offered before completing "Zanzil's Secret" and is still offered after all 3 other quests are completed even though it serves no purpose;
3. The whole implementation of the quest seems to revolve around the idea that you couldn't get another "Fool's Stout" without doing the quest "Fool's Stout" in case you fail, though i have never seen any quest being handled in a way where you would get a provided item only once;
4. This quest is not a repeatable, nor mention to be one, thus you could only get a 2nd "Fool's Stout" once, making the whole idea of this quest a bit redundant as a "If you fail, you need to do this quest" because if you fail twice, you will not be able to progress through the quest-line.

All of this leads to the quests having any of these conditions:
1. "Fool's Stout" needs to be a repeatable quest.
2. If you have any quest that gives you a "Fool's Stout" and is failed. You should not get another "Fool's Stout" by abandoning and retaking it or make the item "Fool's Stout" have the same timer as before when the player abandoned it.
3. "Fool's Stout" shouldn't be offered before you complete "Zanzil's Secret" and should not be offered after you complete either "Get the Gnomes Drunk" or "Get the Goblins Drunk".

Links:
Fool's Stout - http://www.wowhead.com/item=5806
Fool's Stout - http://www.wowhead.com/quest=1127
Zanzil's Mixture and a Fool's Stout - http://www.wowhead.com/quest=1119
Get the Gnomes Drunk - http://www.wowhead.com/quest=1120
Get the Goblins Drunk - http://www.wowhead.com/quest=1121

Core revision: 524152b5bf94
Database version: TDB 335.53
Addons: Anticheat1
##

**Comment by offl** [comment]
@ZenoX92 since you already done some work for this chain maybe you can take a look at this? Because I assume your latest fixes are based on videos, so you saw how these quests work(?) :)

**Comment by ZenoX92** [comment]
This quest is working just fine, @offl. It's repeatable and can only be obtained if player is on quest 'Zanzil's Mixture and a Fool's Stout'... as it should be.
I think someone fixed this and forgot to close the issue (?)

**Comment by Aokromes** [comment]
Maybe it was fixed by someone opening a fix ticket without search xd

**Comment by ZenoX92** [comment]
Yeah, probably... :)
